assets/src/edit-story/components/panels/layer/layerContent.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const LayerBackground = styled.span`
+	opacity: .5;
+`;
+
+const LayerText = styled.span`
+	overflow: hidden;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+`;
+
+function LayerContent( { element } ) {
+	switch ( element.type ) {
+		case 'text':
+			return <LayerText dangerouslySetInnerHTML={ { __html: element.content } } />;
+
+		case 'image':
+			// Disable reason: Well, it's actually an image element, so it's the best description
```

Comment:
We should think about how to make this more accessible though. If you have 5 image elements they would all be described the same.

Reply:
It would make sense to include the layer number perhaps - and maybe the original file name? _"Layer 5: Image element (harrypotter.jpg)"_.

Reply:
This one is still open.

Reply:
Not sure  if  the layer number is really useful. Assistive technologies would add announce the list item number anyway.